src/main/java/net/openhft/chronicle/threads/MediumEventLoop.java

```diff
@@ -411,13 +415,15 @@ protected boolean runAllHandlers() {
                         handleExceptionMediumHandler(handlers[1], e);
                     }
                     // fall through
-                case 1:
+                case 1: {
```

Comment:
we don't do this anywhere else - why add here?

Reply:
Apparently, there are additional requirements on the last "case".